.rubocop.yml

```diff
@@ -12,6 +12,7 @@ AllCops:
   Exclude:
     - "db/migrate/*"
     - "lib/package_builder.rb"
+    - "config/initializers/business_time.rb"
```

Comment:
what are we ignoring here?

Reply:
rubocop complains that 
```
BusinessTime::Config.load("#{Rails.root}/config/business_time.yml")
```
should instead use the Rails.root.join method. But I tried that and the whole thing went crazy because the gem could no longer find the path. So I ignored it instead

Reply:
``` 
Offenses:

config/initializers/business_time.rb:1:27: C: Rails/FilePath: Prefer Rails.root.join('path/to').
BusinessTime::Config.load("#{Rails.root}/config/business_time.yml")
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
when I tried changing it to:
```
BusinessTime::Config.load("#{Rails.root.join('/config/business_time.yml')}")
```
or to
```
BusinessTime::Config.load(Rails.root.join('/config/business_time.yml'))
```
I get this error when running any spec:
```
mreal $ rspec spec/models/planning_application_spec.rb

An error occurred while loading ./spec/models/planning_application_spec.rb.
Failure/Error: require File.expand_path("../config/environment", __dir__)

Errno::ENOENT:
  No such file or directory @ rb_sysopen - /config/business_time.yml
# /Users/mreal/.rvm/gems/ruby-2.6.6/gems/business_time-0.10.0/lib/business_time/config.rb:161:in `initialize'
```

Reply:
ah I got it now, had an extra slash. changing...